swagger/sdrangel/code/qt5/client/SWGHttpRequest.cpp

```diff
@@ -163,7 +163,7 @@ void SWGHttpRequestWorker::execute(SWGHttpRequestInput *input) {
         // variable layout is MULTIPART
 
         boundary = "__-----------------------"
-            + QString::number(QDateTime::currentDateTime().toTime_t())
```

Comment:
... and here: https://github.com/f4exb/swagger-codegen/blob/sdrangel/modules/swagger-codegen/src/main/resources/qt5cpp/HttpRequest.cpp.mustache#L157